Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Multistage docker build, requires docker 17.05
+
+# builder stage
+FROM rust:1.26 as builder
+
+RUN set -ex && \
+    apt-get update && \
+    apt-get --no-install-recommends --yes install \
+        clang \
+        libclang-dev \
+        llvm-dev \
+        libncurses5 \
+        libncursesw5 \
+        cmake \
+        git
+
+WORKDIR /usr/src/grin
+
+# Copying Grin
+COPY . .
+
+# Building Grin
+RUN cargo build --release
+
+# runtime stage
+FROM debian:9.4
+
+RUN set -ex && \
+    apt-get update && \
+    apt-get --no-install-recommends --yes install locales && \
```

Comment:
Dont we still need ncurses library in the runtime image?

Reply:
It looks like we don't need ncurses in the runtime image (tested locally on macOS 10.13.5).